omegalambda/main/controller/focuser_control.py

```diff
@@ -50,7 +50,7 @@ def check_connection(self):
             return True
         else:
             logging.error("Could not connect to focuser")
-            return False
+            return True
```

Comment:
You could just delete return True on like 50 and bring line 53 inline with the if else block.